docs/en/observability/synthetics-visualize.asciidoc

```diff
@@ -1,9 +1,5 @@
 [[synthetics-visualize]]
-= Visualize synthetic monitoring
-
-++++
-<titleabbrev>Visualize</titleabbrev>
-++++
+= Visualize user journeys
```

Comment:
```suggestion
= Visualize journeys
```

seems more consistent with our use of journeys elsewhere. Alternatively, we could use maybe "Browser Journeys"